versioning/scripts/dockerfiles/main.go

```diff
@@ -82,14 +94,20 @@ func verifyDockerfiles(spec versions.Spec, tmpl template.Template) (failureCount
 		path := filepath.Join(version.Dir, "Dockerfile")
 		dockerfile, err := ioutil.ReadFile(path)
 		check(err)
-
 		foundDockerfile[path] = true
 
 		if string(dockerfile) == string(data) {
 			log.Printf("%s: OK", path)
 		} else {
-			failureCount++
-			log.Printf("%s: FAILED", path)
+			// there is chance the the only difference between files are
```

Comment:
nit: I would change the comment to add capitalization and periods and shorten it to:

```
// Ignore differences caused by whitespaces and tabs.
```

Reply:
OK, fixed.